src/config/notifier.ts

```diff
@@ -30,9 +27,16 @@ const notifier = (args: {
   switch (type) {
     case signInRoutine.FAILURE:
     case signUpRoutine.FAILURE:
-    case signInWithPasswordRoutine.FAILURE:
+    case signInWithPasswordRoutine.FAILURE: {
+      if (
+        payload.error instanceof AxiosError &&
+        payload.error.response?.data.statusCode === StatusCodes.NOT_ACCEPTABLE
+      )
+        message = FAILURE_MESSAGES.MEMBER_WITHOUT_PASSWORD;
```

Comment:
Couldn't it be done in query client?

Reply:
@pyphilia Yes, we could have done the test in the query-client and changed the error message just in that case to be the `FAILURE_MESSAGES.MEMBER_WITHOUT_PASSWORD`. Do you think this would be better ?